server/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:alpine

```

Comment:
I've been wanting to keep away from Docker to remove dependencies. Especially with how little is in this Dockerfile, I think it's better to just have people install it natively.